src/NamespaceTracker.hs

```diff
@@ -96,10 +96,13 @@ fanOutBundle i tms = get >>=
   where
     deriveMsgs tms i' ps = tmMessage <$> filter (includeMsg i' ps) tms
     includeMsg i' ps tm =
+        -- NOT SURE THIS IS TRUE!
```

Comment:
Why not?

Reply:
Because we're supposed to forward them to the owner, not the client aren't we?